turn/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG VERSION
+FROM instrumentisto/coturn:${VERSION:-latest}
```

Comment:
This has been deprecated: https://github.com/instrumentisto/coturn-docker-image/commit/0a4a7f0963118d91110c51d49abd88a28c2ba7d9

It is suggested to switch to the upstream image [`coturn/coturn`](https://hub.docker.com/r/coturn/coturn) ([🔗](https://github.com/coturn/coturn/tree/master/docker/coturn)).